### ISSUE TITLE
dedup is not defined

### ISSUE THREAD

**Issue by Covertness** [telehash/telehash-js#46]
When peers link though a router, the router will crash:
```bash
ReferenceError: dedup is not defined
    at Object.receive_channel [as channel] (/Users/yangzhang/Projects/test-telehash/node_modules/telehash/lib/util/receive.js:50:10)
    at Mesh_receive [as receive] (/Users/yangzhang/Projects/test-telehash/node_modules/telehash/lib/mesh.class.js:265:20)
    at /Users/yangzhang/Projects/test-telehash/node_modules/telehash/node_modules/telehash-udp4/index.js:35:12
    at Object.tp.pipe (/Users/yangzhang/Projects/test-telehash/node_modules/telehash/node_modules/telehash-udp4/index.js:52:21)
    at Socket.receive (/Users/yangzhang/Projects/test-telehash/node_modules/telehash/node_modules/telehash-udp4/index.js:34:8)
    at Socket.emit (events.js:110:17)
    at UDP.onMessage (dgram.js:472:8)
```

**Comment by rynomad** [comment]
Good catch, i think this got re-introduced as i was shuffling some branches. Should be fixed in 0.4.0

**Comment by rynomad** [comment]
0.4.0 published with correct pointer to deduper :)
